cmd/dockerd/config_unix.go

```diff
@@ -44,4 +45,6 @@ func installConfigFlags(conf *config.Config, flags *pflag.FlagSet) {
 	flags.Var(&conf.ShmSize, "default-shm-size", "Default shm size for containers")
 	flags.BoolVar(&conf.NoNewPrivileges, "no-new-privileges", false, "Set no-new-privileges by default for new containers")
 	flags.StringVar(&conf.IpcMode, "default-ipc-mode", config.DefaultIpcMode, `Default mode for containers ipc ("shareable" | "private")`)
+	flags.Var(&conf.NetworkConfig.DefaultAddressPools, "default-node-local-address-pools", "Default address pools for node specific local networks")
```

Comment:
I briefly discussed with @mavenugo, and it should be ok to remove the `node-local` from the flag name. The flag should also be singular (each flag sets a single address pool, and to define multiple pools, you have to set the flag multiple times). Inside the `daemon.json` file, the field _should be_ plural though; for that to work, the flag needs to be defined as a "namedListOpt"; see this PR for how that's done: https://github.com/moby/moby/pull/36125; https://github.com/moby/moby/blob/a80cd04eb5375f0eb7b648e820f694b45983cf2b/cmd/dockerd/config.go#L70

Reply:
The format (after `node-local` is removed) is ok; but that's the daemon.json. For the flag name, that should be singular

Reply:
changed the flag name to singular